api/routes.py

```diff
@@ -161,7 +161,15 @@ def convert_json_file_to_csv(self, file_key):
             temp_path = save_file(file_key='file_name')
             return temp_path
         
-
+def parse_bool(str_bool):
```

Comment:
Maybe try casting

Reply:
@milen-sage Per discussion on slack with @andrewelamb, suggestions were to try converting the string entry parameter by using the `.astype()` method in [this way](https://github.com/Sage-Bionetworks/schematic_db/blob/develop/schematic_db/rdb_updater/rdb_updater.py#L144) or in [this way](http://pandas.pydata.org/pandas-docs/stable/reference/api/pandas.Series.astype.html#). We decided that neither of these were optimal as they are methods for `Dataframe` and `Series` objects, and that `string` type objects do not have such a built in method.

We also considered the use of [this method](https://www.educative.io/answers/what-is-the-numpycharchararrayastype-method-in-python), but decided that it was also not the best fit as it was for `numpy.array` objects and not `string`.

Previously, I had tried using `bool('true')` or `bool('false')` to convert the string parameter entry to boolean and found this ineffective as non-empty strings always resolve to `True` and only empty strings resolve to `False`. We tried to see if this behavior was case sensitive and trialed `bool('True')` and `bool("False")`, and found the behavior to be the same as above.

The last option would be to do mapping like in [this example](https://statisticsglobe.com/convert-string-boolean-pandas-dataframe-column-python). We could adapt it to use a dictionary mapping for the one string, but the currently implemented function has the benefit of being accessible to other endpoints that would need booleans parsed, like if we had to add the `restrict_rules` parameter to the `/model/validate` endpoint.

Reply:
Yes, given all, the way you have it works - thank you for looking into this!